EosGround/radioDriver.py

```diff
@@ -0,0 +1,112 @@
+import time
+
+from digi.xbee.devices import XBeeDevice
+from digi.xbee.devices import RemoteXBeeDevice
+from digi.xbee.devices import XBee64BitAddress
+
+from EosLib.packet.packet import Packet
+import EosLib.packet.definitions
+import EosLib.packet.packet
+import EosLib.packet.transmit_header
+
+import psycopg2
```

Comment:
digi-xbee and psycopg2 are missing from the requirements.txt -- run `pip freeze` and copy over the lines for digi-xbee, psycopg2, and pyserial